metadata-ingestion-modules/airflow-plugin/pyproject.toml

```diff
@@ -29,30 +37,21 @@ classes = ["typing"]
 
 [tool.ruff.lint]
 select = [
-    "B",
-    "C90",
-    "E",
-    "F",
-    "I", # For isort
-    "TID",
+    "B",    # flake8-bugbear
+    "C90",  # mccabe complexity
+    "E",    # pycodestyle errors
+    "F",    # pyflakes
+    "I",    # isort
+    "TID",  # flake8-tidy-imports
 ]
 ignore = [
-    # Ignore line length violations (handled by Black)
-    "E501",
-    # Ignore whitespace before ':' (matches Black)
-    "E203",
-    "E203",
-    # Allow usages of functools.lru_cache
-    "B019",
-    # Allow function call in argument defaults
-    "B008",
+    "E501",  # Line length violations (handled by formatter)
```

Comment:
also want to make sure you've seen https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules